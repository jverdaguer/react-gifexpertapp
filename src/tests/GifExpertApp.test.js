import { shallow } from 'enzyme';
import GifExpertApp from '../GifExpertApp';

describe('Test related to <GifExpertApp />', () => {
    test('should mount component <GifExpertApp />', () => {
        const wrapper = shallow(<GifExpertApp />);
        expect(wrapper).toMatchSnapshot();
    });
    test('should show default Categories', () => {
        const defaultCategories = ['One Punch', 'He Man'];
        const wrapper = shallow(
            <GifExpertApp defaultCategories={defaultCategories} />
        );
        expect(wrapper.find('GifGrid').length).toBe(defaultCategories.length);
    });
});
