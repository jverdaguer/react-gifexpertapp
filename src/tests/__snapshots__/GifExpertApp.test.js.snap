// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test related to <GifExpertApp /> should mount component <GifExpertApp /> 1`] = `
<Fragment>
  <h2>
    GifExpertApp
  </h2>
  <AddCategory
    setCategories={[Function]}
  />
  <hr />
  <ol />
</Fragment>
`;
